test/one/profiler/test/Output.java

```diff
@@ -51,6 +60,19 @@ public double ratio(String regex) {
         return (double) matched / total;
     }
 
+    public String convert(ProfileOutputType fromOutputType, ProfileOutputType toOutputType, InputStream inputStream) {
```

Comment:
Why accepting `InputStream` and producing `String`?
The source for the conversion is already available in this class as `lines` field. The destination of the conversion is another `Output` instance. So, we convert this `Output` to a new `Output`.